applauncher/004-three-by-three.qml

```diff
@@ -204,9 +145,60 @@ Item {
                 }
             }
         }
+    }
+
+    Component {
+        id: spacer
+        Item {
+            height: root.height / numColumns / 2
```

Comment:
While we are at it, let's make this height conditional on whether the screen is round or not.

Design decision ratified by @eLtMosen :p

Reply:
Or maybe we should have a similar margin as the GridView (`parent.width * .026` ?) idk, whatever looks best.

Reply:
> While we are at it, let's make this height conditional on whether the screen is round or not.
> 
> Design decision ratified by @eLtMosen :p

Indeed, i was only made aware by @FlorentRevest that it might make sense to have no bottom margin on square screens and thus to show all 3 rows. Fully agree on that and still question myself for not having noticed while testing on sparrow. `/o\`

Reply:
Done!  I tried it on `dory` (my only square watch) and it looks good.